spec/factories/oidc_user_factory.rb

```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :oidc_user do
+    sub { "user-id" }
```

Comment:
`sub` is unique so this should be `sequence(:sub) { |n| "user-id-#{n}" }`

Reply:
Great improvement, thanks!